### ISSUE TITLE
No theme found in context

### ISSUE THREAD

**Issue by goottime** [sunarya-thito/shadcn_flutter#22]
I installed version [b1a3ba6](https://github.com/sunarya-thito/shadcn_flutter/commit/b1a3ba6bae3e04b57321e1a22cbf2711757e5087) of the library directly from github and got the following error when running it:

![image](https://github.com/user-attachments/assets/cbf574d5-1000-4784-a6a2-c3412cdeee5e)

My startup code looks like this:
```
  @override
  Widget build(BuildContext context) {
    return ShadcnApp(
      title: 'My App',
      home: AccountPage(),
      theme: ThemeData(
        colorScheme: ColorSchemes.darkZinc(),
        radius: 0.7,
      ),
    );
  }
```

**Comment by sunarya-thito** [comment]
Can you confirm if this is still happening in the latest commit?

**Comment by sunarya-thito** [comment]
Should be solved with [5f8582a](https://github.com/sunarya-thito/shadcn_flutter/commit/5f8582acad06c4624a301f394d69eaeaa7cb9fd2)
